src/crawlers/node.js

```diff
@@ -42,7 +42,7 @@ function nodeRecReadDir(roots, {ignore, exts}) {
           }
 
           if (filePath.match(extPattern)) {
-            retFiles.push(filePath);
+            retFiles.push(path.resolve(filePath));
```

Comment:
This particular one isn't avoidable. If I remove `path.resolve()`, the file path under windows comes back without the drive letter and is unusable :(

I was able to remove the other case under isDescendant in fastfs though.